subprojects/shipkit/src/main/groovy/org/shipkit/gradle/configuration/ShipkitConfiguration.java

```diff
@@ -255,10 +255,13 @@ public void setIgnoreCommitsContaining(Collection<String> commitMessageParts) {
 
         /**
          * Set the Publication Repository where we look for your published binary.
```

Comment:
Let's write what do we use it for:
 - release notes generation
 - getting binaries for comparing with previous version